test/e2e/flowaggregator_test.go

```diff
@@ -631,6 +662,40 @@ func checkRecordsForToExternalFlows(t *testing.T, data *TestData, srcNodeName st
 	}
 }
 
+func checkRecordsForFlowsAfterRestart(t *testing.T, data *TestData, srcNodeName string, srcPodName string, srcIP string, dstIP string, dstPort int32, isIPv6 bool) {
+	var cmd string
+	if !isIPv6 {
+		cmd = fmt.Sprintf("wget -O- %s:%d", dstIP, dstPort)
+	} else {
+		cmd = fmt.Sprintf("wget -O- [%s]:%d", dstIP, dstPort)
+	}
+	// For the restartFlowAggregator test, we create traffic in every poll as we
+	// do not know exactly when the connection is re-established from the Flow
+	// Exporter to the Flow Aggregator. We will modify the test when there is
+	// visibility into connection re-establishment in the future.
+	var recordToCheck string
+	err := wait.PollImmediate(2*time.Second, restartFlowAggregatorTimeout, func() (bool, error) {
```

Comment:
would it be ok to call `checkRecordsForToExternalFlows` within the `Poll`? I don't understand why the logic has to be so different.

Reply:
If calling `checkRecordsForToExternalFlows` within the `Poll`, it will requires many logic change in `checkRecordsForToExternalFlows` and `getCollectorOutput` because for every error we need to return `false, error` for `wait` to work. The `Poll` here is different because we don't want to check re-connection through logs, so it needs to take longer and periodically create traffic to get records delivered to new Flow Aggregator Pod. Or I can make https://github.com/antrea-io/antrea/pull/2668 ready before we merge this PR then it will be more clear.

Reply:
I would suggest: 1) merging this PR without the new test case (the change is pretty simple and LGTM), 2) merging #2668, 3) writing a "cleaner" testcase

Reply:
Sounds good to me. I will create a new testcase when #2668 gets merged. Thanks!